azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/scheduledflowpage.vm

```diff
@@ -114,17 +121,28 @@
                 #else
                   <td>Not Applicable</td>
                 #end
-                <td><button type="button" data-toggle="modal" data-target="#executionOptions">show</button></td>
+                <td>
+                ## Changed the style of "Show" button to be consistent with other buttons.
+                  <button type="button" class="btn btn-sm btn-info" data-toggle="modal"
+                          data-target="#executionOptions-${velocityCount}">Show
+                  </button>
+                </td>
 
-                  <div class="modal fade" id="executionOptions" tabindex="-1" role="dialog" aria-labelledby="executionOptionsLabel">
+              ## Distinguished executionOptions of each schedule with unique count number to fix
```

Comment:
Could you elaborate more on how the unique count works?

Reply:
The original issue was that all the schedules shared the same data-target id. I added "velocityCount" to each id because it is basically the index of each schedule and it's unique, so it can be used to identify each data-target.

Reply:
Where is the code that parses the executionOptions-<number>?

Reply:
We first call handleGetAllSchedules to obtain all schedules. Then for the executionOptions in each schedule, we tag it with a unique id called "executionOptions-${velocityCount}", and then display it when clicking the show button. Not sure If I answered your question?